Simplenote/ActivityIndicatorAlertViewController.swift

```diff
@@ -0,0 +1,68 @@
+import UIKit
+
+class ActivityIndicatorAlertViewController: UIViewController {
```

Comment:
I got confused by `AlertView` in the name :-)

Reply:
Yup! How about `SpinnerViewController` or something like that?

Reply:
Works for me :)